src/main/java/edu/byu/cs/autograder/TrafficController.java

```diff
@@ -64,9 +64,15 @@ public void addGrader(Grader grader) {
         executorService.submit(grader);
     }
 
-    public void notifySubscribers(String netId, Map<String, Object> message) {
-        sessions.get(netId).stream()
-                .filter(Session::isOpen)
-                .forEach(session -> WebSocketController.send(session, message));
+
+    public synchronized void notifySubscribers(String netId, Map<String, Object> message) {
```

Comment:
Now that we aren't using streams anymore, this doesn't need to be synchronized

Reply:
Hopefully haha

We'll see how it performs without it in production :)

Reply:
See the conversation in #142 and the changes intentionally introduced in #138.

Reply:
change has been made, and appears to still function correctly

Reply:
@DallinFromEarth Did you forget to push your changes? The function is still showing as `synchronized` as of commit `e45ca46`.

Reply:
changes have been made on commit [97d9a26](https://github.com/softwareconstruction240/autograder/pull/145/commits/97d9a2646454cd192039ebc07c95241ae85fd74d)

Reply:
@frozenfrank changes have been made on commit [97d9a26](https://github.com/softwareconstruction240/autograder/pull/145/commits/97d9a2646454cd192039ebc07c95241ae85fd74d), pushed several hours ago

Reply:
commit `e45ca46` was the first commit I made, before pushing onto this branch or making the pull request